src/Aquifer.API/Endpoints/Resources/Content/List/NeedsTranslation/Endpoint.cs

```diff
@@ -0,0 +1,135 @@
+using Aquifer.API.Common;
+using Aquifer.API.Services;
+using Aquifer.Common.Utilities;
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using FastEndpoints;
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.API.Endpoints.Resources.Content.List.NeedsTranslation;
+
+public class Endpoint(AquiferDbContext dbContext, IUserService userService) : Endpoint<Request, Response>
+{
+    public override void Configure()
+    {
+        Get("/resources/content/needs-translation");
+        Permissions(PermissionName.ReadResources);
+    }
+
+    public override async Task HandleAsync(Request req, CancellationToken ct)
```

Comment:
This is largely a copy of `Resources.Content.List` but was different enough to be it's own thing and not try to factor out any shared logic.